packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -275,17 +272,16 @@ async function parseRequestObject(requestObject: IDataObject) {
 			if (requestObject.formData instanceof FormData) {
 				axiosConfig.data = requestObject.formData;
 			} else {
-				axiosConfig.data = createFormDataObject(requestObject.formData as object);
+				axiosConfig.data = createFormDataObject(requestObject.formData as Record<string, unknown>);
 			}
 			// Mix in headers as FormData creates the boundary.
 			const headers = axiosConfig.data.getHeaders();
 			axiosConfig.headers = Object.assign(axiosConfig.headers || {}, headers);
 			await generateContentLengthHeader(axiosConfig.data, axiosConfig.headers);
 		} else if (requestObject.body !== undefined) {
 			// If we have body and possibly form
-			if (requestObject.form !== undefined) {
+			if (requestObject.form !== undefined && requestObject.body) {
 				// merge both objects when exist.
-				// @ts-ignore
 				requestObject.body = Object.assign(requestObject.body, requestObject.form);
```

Comment:
We should change this to either
`Object.assign(requestObject.body, requestObject.form)` or `requestObject.body = Object.assign({}, requestObject.body, requestObject.form)`

we should also perhaps (eventually) create a linting rule for `Object.assign` should be used.

Reply:
This pattern occurs very frequently elsewhere. Also okay if we fix this in full but separately through the lint rule to be written?

Reply:
okay